datacontract/data_contract.py

```diff
@@ -67,8 +80,9 @@ def lint(self) -> Run:
                 result="passed",
                 name="Data contract is syntactically valid",
                 engine="datacontract"
-            ))
-            run.checks.extend(ExampleModelLinter().lint(data_contract))
+                ))
+            for linter in self.enabled_linters:
+                run.checks.extend(linter.lint(data_contract))
```

Comment:
When a linter is broken, the whole linting process stops. We could have an `except` clause in here to cater for this.

Reply:
Fixed.